code_share/app_code_share/views.py

```diff
@@ -26,7 +26,7 @@ def home(request):
     """
 
     if request.method == 'GET':
-        return render(request, 'app_code_share/homepage.html', {})
+        return render(request, 'homepage.html', {})
```

Comment:
so technically code_share is your main app_name_folder/templates :) and the rest of the things such as wsgi etc. remains outside hence I have kept it to this. Also I have made changes in `settings.settings` in STATIC_DIRS part to accomodate the change I've done :)

It would be good to have all static files in one place :)

Reply:
your take on this one? @sourabhtk37

Reply:
Yeah, the changes you made are good, global template file with sub folders and configuration related files in root dir and all. 

What I requested to changed is this:
`'DIRS': ['code_share/templates/app_code_share', 'code_share/templates/error', ],`

If we changed to having `'code_share/templates/'` only, then writing in views something like `return render(request, 'homepage.html', {})` would become `return render(request, 'app_code_share/homepage.html', {})` which is better to find the file. Just my opinion since in case we have many sub folders it becomes quite hard to find for new contributors to find the file.

Reply:
Hmmm.. makes sense. Let me go through it. Will it push it by tonight :)